src/sdk.ts

```diff
@@ -86,9 +75,20 @@ export function init<O>(
     finalOptions.transport = NativeTransport;
   }
 
-  originalInit(finalOptions);
+  const browserOptions = {
+    ...finalOptions,
+    autoSessionTracking:
+      NATIVE.platform === 'web' && finalOptions.enableAutoSessionTracking,
+  } as O;
+
+  const mobileOptions = {
+    ...finalOptions,
+    enableAutoSessionTracking:
+      NATIVE.platform !== 'web' && finalOptions.enableAutoSessionTracking,
+  } as CapacitorOptions;
 
-  void NATIVE.initNativeSdk(finalOptions);
+  originalInit(browserOptions);
+  void NATIVE.initNativeSdk(mobileOptions);
```

Comment:
Do we call this for web?

Reply:
Yep, and if `enableNative = false` it sets `NATIVE.enableNative` to false and then the native wrapper does not do anything.